cypress/e2e/api/posts/createNewPost.cy.js

```diff
@@ -0,0 +1,21 @@
+import post from "../../../fixtures/newPost.json";
+
+beforeEach(() => {
+  cy.api({
```

Comment:
You are reassigning value to alias and it is not obviously on my opinion. I would not not use hooks this way

Reply:
better create function or custom command and reuse it

Reply:
but it is not required for this home work